docs/developing.md

```diff
@@ -135,6 +135,31 @@ Alternatively, you can run the Dep command directly.
 dep ensure -v
 ```
 
+## Using Serving Certificates
+
+The hiveutil command includes a utility to generate Letsencrypt certificates for use with clusters you create in Hive.
+
+Prerequisites:
+* The `certbot` command must be available and in the path of your machine. You can obtain by following instructions at:
+  [https://certbot.eff.org/docs/install.html](https://certbot.eff.org/docs/install.html)
+* You must have credentials for AWS available in your command line, either by a configured ~/.aws/credentials or environment variables.
+
+### Generating a Certificate
+
+1. Ensure that the hiveutil binary is available (`make hiveutil`)
```

Comment:
s/hiveutil/`hiveutil`/ , usually it is better to keep the binary names highlighted.